### ISSUE TITLE
[0.9.1-ish] Ever increasing "updated peer set"

### ISSUE THREAD

**Issue by shish** [influxdb/influxdb#3395]
Every time I restart influxdb I get an additional

`[metastore] 2015/07/08 23:56:46 [DEBUG] raft: Node 127.0.0.1:8088 updated peer set (2): [127.0.0.1:8088]`

(I wonder if this is related to the IO load and write-timeouts in #3275, if every write is being multiplied by each copy of localhost...)

**Comment by shish** [comment]
I deleted the raft.db file and now this message only appears once, but I also get:

```
[graphite] 2015/07/20 21:47:13 failed to write point batch to database "graphite": at least one node required
[graphite] 2015/07/20 21:47:13 failed to write point batch to database "graphite": at least one node required
[graphite] 2015/07/20 21:47:13 failed to write point batch to database "graphite": at least one node required
[graphite] 2015/07/20 21:47:13 failed to write point batch to database "graphite": at least one node required
[...]
```

**Comment by jwilder** [comment]
The original log message is just debug logging from the raft library.  It does not log that in 0.9.3 unless you enable cluster tracing now.

Deleting raft.db on a single node setup will break things in unknown ways.  You will need to recreate that cluster.

**Comment by shish** [comment]
Even if it doesn't *log* that the database is growing, the database is still growing, no? It's the growth that concerns me, not the log messages themselves...

**Comment by shish** [comment]
For reference, my raft.db now contains 4191 instances of "127.0.0.1:8088" (I need to restart the daemon every few minutes because of #3275, and every restart adds an instance). Removing the logging does mean that my logs will be 95% smaller since I'm not writing 4000 copies of "Node 127.0.0.1:8088 updated peer set" into the file on every restart, at least...

**Comment by jwilder** [comment]
The IO load and timeouts are related to the WAL and are being addressed in #3514.  The raft logging and clustering has been changed significantly since 0.9.1 so you'll probably want to upgrade to 0.9.3 to resolve these issues.
